src/app/dataValidator/rules/topstories.js

```diff
@@ -0,0 +1,55 @@
+import deepClone from '../../helpers/json/deepClone';
```

Comment:
Not convinced this is homed in the right place. `src/app/dataValidator/` is only for the dataValidation logic which is checking the Optimo payload format on the local dev env. Realistically in the future we will remove it from simorgh (_reminds me that I need to write that issue_). 

Whereas this is actual production code for flattening the payload after `getInitialData` and should live somewhere more suitable such as:
- `src/app/lib/`

Reply:
I've since realised that there is also timestamp logic homed in the same place as this. I'd be happy for the outcome to be that we create an issue to move both of these out of the `dataValidator/` directory rather than the change being in this PR.

Reply:
Done https://github.com/bbc/simorgh/issues/1616